data-model/SDP-data-model-templates/textual-artifact_sdp-template.json

```diff
@@ -6,7 +6,7 @@
     "array of strings"
   ],
   "extent": "",
-  "work": "URI",
+  "embodies": "URI of Work Authority File",
```

Comment:
Trying this out. In keeping with the way we've done other relationships, I've changed the field name to be the 'property'/'predicate'. For reference, this term is used in the Linked Ancient World Ontology to connect "Written Works" to "Conceptual Works" (i.e., a written work embodies a conceptual work).